genai/api/genai_api/src/main.py

```diff
@@ -306,6 +343,42 @@ def genai_image(payload: Payload_Image):
         )
 
 
+@app.post("/genai/npc_chat", tags=["npc_chat"])
+def genai_npc_chat(payload: Payload_NPC_Chat):
+    try:
+        request_payload = {
+            'message': payload.message,
+            'from_id': payload.from_id,
+            'to_id': payload.to_id,
+            'debug': payload.debug
+        }
+        logging.debug(f'request_payload: {request_payload}')
+        response = requests.post(f'{GENAI_NPC_CHAT_ENDPOINT}', headers=headers, json=request_payload)
+        logging.debug(f'response: {response}')
+        logging.debug(f'response type: {type(response)}')
+        logging.debug(f'response.content: {response.content}')
```

Comment:
for consistency I could take out the logging.debugs here (leave the payload)

Reply:
I had it in because I was having issues with the `genai = "GKEGenAI"` not working. I'll go ahead and remove it since that appears to be a separate issues, because this code works perfectly fine when `genai = "VertexAI"`.